app/views/curation_concerns/base/_form_child_work_relationships.html.erb

```diff
@@ -0,0 +1,65 @@
+<%# Form UI behavior code and details;
+Code:
+  app/assets/javascripts/sufia/relationships_table.es6
+CSS:
+  table.relationships-ajax-enabled : Used by sufia JS app as selector to wire up the UI functionality
+  tr.new-row : The basic template row for cloning when user clicks "Add"
+  .btn-remove-row : Button to remove its parent TR from the table
+  .btn-add-row : Button to clone its parent TR and inject a new row into the table
+  input.new-form-control : Input field for client client side validation and additional features
+  .message.has-warning : Used to display UI errors related to input values and server errors
+HTML Properties:
+  table:
+    data-query-url : URL base to append newly added work ID to for querying details (title, href)
+%>
+<div class="form-group multi_value optional managed">
+  <%= link_to "Attach New Work", polymorphic_path([main_app, :new, :curation_concerns, :parent, curation_concern.model_name.singular], parent_id: params[:id]), target: "_blank", class: 'btn btn-primary' %>
+
+  <table class="table table-striped related-files relationships-ajax-enabled"
+         data-query-url="<%= polymorphic_path([main_app, :curation_concerns, curation_concern.model_name.singular], id: '$id') %>">
+    <thead>
+    <tr>
+      <th>Child Work</th>
+      <th>Actions</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr class="hidden">
+      <td>
+        <% curation_concern.ordered_members.to_a.select { |k| k.class.to_s == "FileSet" }.each do |fileset| %>
+          <input class="related_works_ids generic_work_filesets_ids" name="generic_work[ordered_member_ids][]" type="hidden" value="<%= fileset.id %>">
+        <% end %>
+      </td>
+      <td></td>
+    </tr>
+    <% curation_concern.ordered_members.to_a.select { |k| k.class.to_s != "FileSet" }.each do |member| %>
+        <tr>
+          <td>
+            <%= link_to "#{member.title.first}", polymorphic_path([main_app, :curation_concerns, curation_concern.model_name.singular], id: member.id) %>
```

Comment:
1) Why are you interpolating the string rather than just `member.title.first`
2) Can you just `link_to title, member`?